### ISSUE TITLE
The VMs are not on the same port group when connecting the CVCs to one vlan service in parallel (API)

### ISSUE THREAD

**Issue by DoronFried** [QualiSystems/vCenterShell#307]
Related to Test Case 151080.

STR:
1. have a reservation with 4 D.Apps and one Auto vlan service (don't insert any Vlan ID)
2. Add CVC Request between the 4 D.Apps to the Vlan service (4 CVC request are added)
3. open studio and connect the 4 CVC requests in Parallel (use the parallel tool and the method ConnectRoutesInReservation) 

BIR:
if The Vms are not in the Same Vlan ID (port group).
-> in my case one VM was on port group "Vlan 18" and the others 3 Vms were on Port Group "Vlan 19". also the Vlan service attribute is on "Vlan 19"

**Comment by alonagetzler** [comment]
Please check without using the Parallel tool, just one call of ConnectRoutesInReservation for all 4 connection. if the bug doesn't reproduce in this scenario please lower priority.

**Comment by yoavEkshtein** [comment]
verified on 1.0.176
